games-board/gnome-mines/gnome-mines-48.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..13} )
+inherit gnome.org gnome2-utils meson python-any-r1 vala xdg
+
+DESCRIPTION="Clear hidden mines from a minefield"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-mines"
+
+LICENSE="GPL-3+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+
+RDEPEND="
+	>=dev-libs/glib-2.40:2
+	dev-libs/libgee:0.8
+	>=x11-libs/gtk+-3.24:3
```

Comment:
It was ported to gtk4
-gtk_dep = dependency ('gtk+-3.0', version: '>= 3.24')
+gtk_dep = dependency ('gtk4', version: '>= 4.6')